### DISCUSSION TITLE
Safety of home heating when using ESPAltherma

### DISCUSSION THREAD

**Discussion by hansons83** [raomin/ESPAltherma#254] [Q&A]
Hi.
I started using ESPAltherma to monitor and control my daikin heat pump. I have a problem with house overheating during sunny days which i try to solve with this great project as i don't use any room thermostat. Initial tests looks great i can make temperature calculation as complicated as i need with home assistant and control ESPaltherma switch based on that, but i have one concern. If i understand code correctly there is no any safety feature in case mqtt connection is lost or home assistant is not working? I have in mind feature which make sure heating is ON when there is no option to control ESPaltherma switch. I think it is probable scenario in which heating may be off for many hours/days.

**Comment by fellek** [comment]
hi, 
overheating seems to be a problem of your configuration. try to change your settings on your indoor unit user interface. 
My preferred configuration is in a well isolated house with outside temp measurement on outside unit at the rooftop.
Some values might be different to your installation, because of colder winter or other variables.

(Installer settings)
```
A.2.1 Standard
Unit control method: LWT
Pump Operation: Sample
```

```
A.3.1 Leaving water > 
A.3.1.1 Main

LWT setpoint mode: WD auto/scheduled
Temperatur range: [max heating lower than now]
Set weather-dependent heating: [as you wish]
```

```
A.3.3 Operation range
Space heating OFF: 15°C
```
(means: if outside Temp is above do not heat!) if you live in other environment/isolated house you might to change to higher or lower Temperature.

```
A.6 System operation
Averaging time: 24h
```

You are responsable for your changes to your devices! only do changes if you know what you are doing and you are able to recover them, if changes do not bring the desired effect!
